src/js/block-visibility/editor.scss

```diff
@@ -1,34 +1,45 @@
 /**
  * Editor Styles for block visibility Inspector Components
  *
- * @since    1.0.0
- * @version  1.6.0
+ * @since 1.0.0
+ * @version [version]
  */
 
-.llms-block-enrollment-visibility {
+.llms-block-visibility {
+    margin-left: auto;
+    margin-right: auto;
+    max-width: 630px;
+	position: relative;
 
-	display: none;
+	&:before {
+		border: 1px solid #e0e0e0;
+		bottom: -6px;
+		content: '';
+		left: -6px;
+		position: absolute;
+		right: -6px;
+		top: -6px;
+	}
 
-	background: $color-grey-light-extra;
-	color: $color-editor-text;
-	left: -14px;
-	line-height: 1;
-	margin-bottom: 5px;
-	padding: 1px;
-	position: absolute;
-	top: -14px;
+	.llms-block-visibility--indicator {
+		color: $color-editor-text;
+		border-top: 1px solid #e0e0e0;
+		margin-top: -22px; // A 28px margin (from WP core spacers) - 6px spacer.
```

Comment:
I'm not totally sure about this, because it creates this weird case, for "blocks rendered as empty", using 2021:
![Schermata 2021-03-02 alle 16 27 45](https://user-images.githubusercontent.com/7689242/109671767-a9cae000-7b74-11eb-90b8-fc0269cd27c3.png)

But actually we're going to avoid that case, right? 
The block will be always rendered now that we have a different way to notify its visibility... right?

Reply:
What block is this?

Reply:
I've never seen "Block rendered as empty" before

Reply:
Sorry, it's the selected one - so that you could see it on the block's settings, that's why I didn't go into details:
Course Syllabus 
with:
- Display to: non enrolled users or visitors
- Not enrolled in: in all of the selected courses or memberships
- and no courses/memberships selected.

Reply:
Okay thanks... I haven't ever noticed this before but I suppose this is something of a pre-existing issue and what you're taking note of is how it visually overlaps with the empty block content.

I think we should combat this by ensuring that our rendering filters don't run during the REST requests to render the dynamic blocks within the editor. If the rest request is for edit context of a valid user we should skip the filters that detmine whether or not the block should be output. EG: We should always show the full block in edit context.

I've worked out the code already but I'm going to write some tests and push an update in a bit

Reply:
@thomasplevy yes, ok I took it for granted that I was talking about the overlapping because I added the note to the specific CSS rule that was causing it. But of course "reviewing" is not like "reading a review" so yeah I had to be more precise, sorry.

> EG: We should always show the full block in edit context.

Agreed, that's what I meant with:
_The block will be always rendered now that we have a different way to notify its visibility... right?_

:D

Reply:
I think this takes care of the issue: https://github.com/gocodebox/lifterlms-blocks/pull/81/commits/d696e7ccd78689f6a0cf09abf229a75e9c557613

Reply:
see https://github.com/gocodebox/lifterlms-blocks/pull/81#commitcomment-47753734